convert.lua

```diff
@@ -113,30 +113,6 @@ function is_past_convert_msg(key, pkt_num)
            pkt_num > convert_end_pkt_num[key]
 end
 
--- Assume that any TCP stream that starts with a SYN with a payload carries
--- Convert data.
-function is_convert_syn()
-    return tostring(tcp_syn_f()) == '1' and
-           tostring(tcp_ack_f()) == '0' and
-           tostring(tcp_len_f()) ~= '0'
-end
-
--- Assume that any TCP stream that has an ACK with a payload carries Convert
--- data.
-function is_convert_ack()
```

Comment:
maybe best to squash all commits into one → you add code you remove just after without explaining the reason in the commit message